src/Controller/Organization/Planning/PlanningController.php

```diff
@@ -30,7 +33,7 @@ public function __construct(
         $this->planningDomain = $planningDomain;
     }
 
-    public function __invoke(Request $request): Response
+    public function __invoke(Request $request, CacheStrategyInterface $cacheStrategy, CacheItemPoolInterface $cacheTwig): Response
```

Comment:
you should maybe inject those dependencies into the contructor instead of the method, as we already have some injected services there